### PR TITLE
create wrraper

### PR DESCRIPTION
**Pull request by hrechanyktolia**

### CODE REVIEW

File: frontend/src/components/Wrraper.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Wrapper = ({ children }) => {
+  const wrapperStyle = {
```
Comment (FortizFil) [inline frontend/src/components/Wrraper.jsx:4]: We do not write styles in the component, put them in the styles file

File: frontend/src/components/Wrraper.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+  };
+
+  const containerStyle = {
```
Comment (FortizFil) [inline frontend/src/components/Wrraper.jsx:10]: We do not write styles in the component, put them in the styles file

File: frontend/src/components/Wrraper.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    width: '1180px',
```
Comment (FortizFil) [inline frontend/src/components/Wrraper.jsx:11]: max-width would be more correct

**Review by FortizFil — CHANGES_REQUESTED** [review]

**Review by FortizFil — APPROVED** [review]
Bring the stylesheet to a common look and merge

**Comment by hrechanyktolia** [comment]
> Bring the stylesheet to a common look and merge

I don't fully understand, what I have to do. Please explain in more detail

**Pull request merged**